### DISCUSSION TITLE
Passphrase in another language

### DISCUSSION THREAD

**Discussion by Ferywir** [dani-garcia/vaultwarden#3268] [Q&A] [answered]
Hello everyone,

thank you dani-garcia for Vaultwarden, it works great!

However, I have a question, I am French-speaking, for that no problem the web-app or the plugins or application is good in French. On the other hand when I want to generate a secret phrase (passphrase). he only offers me English.

is it possible to change this?

THANKS
Cheers
Ferywir

**Comment by BlackDex** [comment, marked as answer]
No it isn't, this is all client based. And as far as i know they do not have other sample files then English

**Reply by Ferywir** [reply]
Hi,
Thank you for your answer ! ok, I understand.

maybe a point of improvement for future updates :-)!

Have a good day

**Reply by BlackDex** [reply]
If, then it works be needed one the Bitwarden client side. I even think there never been feature requests regarding this already. It's not something to be done on the server side.

**Reply by Ferywir** [reply]
ah so even "official" bitwarden is in English? ok i thought it was only vaultwarden who had the passphrase in english.

so indeed, I can wait a long time :-)

THANKS

**Reply by stefan0xC** [reply]
> If, then it works be needed one the Bitwarden client side. I even think there never been feature requests regarding this already. 

I've found [this thread in the community forum](https://community.bitwarden.com/t/choose-the-word-list-for-passphrases/3372) for this request. So it seems to have been requested for a long time.
